opentelemetry-semantic-conventions/src/opentelemetry/semconv/Http_attributes.py

```diff
@@ -0,0 +1,123 @@
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-lines
+
+from enum import Enum
+
+class HttpAttributes:
```

Comment:
I noticed we've removed the old HTTP semantic convention attributes like `HTTP_METHOD`. Shouldn't these continued to be generated as well?

Reply:
we didn't - they are still where they used to be - https://github.com/open-telemetry/opentelemetry-python/blob/9502a9b436a3e3566ee3a8a419ee56f2a6c80ba8/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/__init__.py

you can keep them for as long as you want.